docs/entity/features.md

```diff
@@ -65,6 +65,34 @@ const Order =
     ...
 })
 ```
+It is possible to access the fields metadata of an entity through the `schema.fields` static property:
```

Comment:
Please, move this content to `#metadata` section (https://herbsjs.org/docs/entity/features#metadata). Since it is a evolution of the current method to retrieve metadata, please, change/refactor the current one.

Reply:
How the current way to get the metadata should work in the new refactory?

Reply:
The old one should be replaces to the new one

Reply:
Updated!

Reply:
Should I remove the `meta.schema` from the BaseEntity in the`gotu` repository?

Reply:
> Should I remove the meta.schema from the BaseEntity in thegotu repository?
No, because it would break the Herbs ecosystem